apps/services/handlers.py

```diff
@@ -257,14 +257,14 @@ def calculate_total_hostel_accommodation_cost(cls, month_price: Decimal, month_d
     async def __create_user_document(cls, session, **kwargs):
         service_id = kwargs.get("service_id")
         name = await cls.__generate_user_document_name(service_id, session)
-        date_created = datetime.strptime(datetime.now().strftime(Settings.DATETIME_FORMAT),
-                                         Settings.DATETIME_FORMAT)
-        kwargs["date_created"] = date_created
+        created_at = datetime.strptime(datetime.now().strftime(Settings.DATETIME_FORMAT),
+                                       Settings.DATETIME_FORMAT)
+        kwargs["created_at"] = created_at
         content = await cls.__create_user_document_content_hostel_settlement_service(**kwargs)
         user_document_record = await user_document_service.create(
             session=session,
             data={
-                "date_created": date_created,
+                "created_at": created_at,
```

Comment:
re-test please and check that datetime is saved in correct format

Reply:
Fixed and tested